spring-cloud-gcp-pubsub/src/main/java/com/google/cloud/spring/pubsub/support/DefaultSubscriberFactory.java

```diff
@@ -359,9 +359,10 @@ SubscriberStubSettings buildGlobalSubscriberStubSettings() throws IOException {
 			subscriberStubSettings.pullSettings().setRetrySettings(retrySettings);
 		}
 
-		if (this.retryableCodes != null) {
-			subscriberStubSettings.pullSettings().setRetryableCodes(
-					this.retryableCodes);
+		Code[] codes = this.retryableCodes != null ? this.retryableCodes
```

Comment:
`this.retryableCodes` is never getting set now, is it, now that the logic for setting it is removed from `GcpPubSubAutoConfiguration`? The global settings are calculated in PubSubConfiguration now.

Reply:
This logic is to cater for `setRetryableCodes` being public. There may still be a possibility of `DefaultSubscriberFactory.setRetryableCodes` being called directly? However, given that retryableCodes is pretty new, the likelihood of this happening may be rare.

Reply:
Makes sense.